client/src/modules/vouchers/toolkit/advances_loans_installments/advances_loans_installments.modal.html

```diff
@@ -0,0 +1,51 @@
+<div class="modal-header">
+  <ol class="headercrumb">
+    <li class="static" translate>TREE.FINANCE</li>
+    <li class="title" translate>VOUCHERS.GLOBAL.ADVANCES_LOANS_INSTALLMENTS</li>
+  </ol>
+</div>
+
+<form name="ToolForm" bh-submit="ToolCtrl.import(ToolForm)" novalidate>
+  <div class="modal-body">
+    <!-- cashbox -->
+    <bh-cashbox-select
+      cashbox-id = "ToolCtrl.searchQueries.cashbox_id"
+      name = "cashbox_id"
+      on-select-callback = "ToolCtrl.onSelectCashbox(cashbox)"
+      restrict-to-user = "false">
```

Comment:
![image](https://user-images.githubusercontent.com/896472/152366902-8ff18228-26f8-4ed2-a041-c3d2f70d249d.png)

This should have `required` set on it, otherwise, it says "optional" at the top.

Reply:
Also, is there any way to make sure we never use the `auxiliary` cashbox?